### ISSUE TITLE
Error on admin config page

### ISSUE THREAD

**Issue by smmsadrnezh** [ComboStrap/combo#34]
Whenever I open ?do=admin&page=config, it prints this in my apache error logs:

```
Got error 'PHP message: PHP Warning:  Declaration of renderer_plugin_combo_renderer::header($text, $level, $pos) should be compatible with Doku_Renderer_xhtml::header($text, $level, $pos, $returnonly = false) in lib/plugins/combo/renderer/renderer.php on line 95PHP message: PHP Warning:  Declaration of renderer_plugin_combo_xml::header($text, $level, $pos) should be compatible with Doku_Renderer_xhtml::header($text, $level, $pos, $returnonly = false) in lib/plugins/combo/renderer/xml.php on line 45', referer: ?do=admin&page=config
```

**Comment by gerardnico** [comment]
Thanks.

It's not an error. It's a warning. The page should show.

This is because you are using a php version that is enforcing type (above 7.2 ?) with a not too old dokuwiki.

The next combo version will be only available on php 7.4 and this warnings should gradually vanish.

**Comment by smmsadrnezh** [comment]
Thanks for your answer. You are right. After upgrading, this warning vanished.
